src/elements/embed/EmbedSpec.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { createCustomField } from "../../plugin/fieldViews/CustomFieldView";
+import { createDefaultRichTextField } from "../../plugin/fieldViews/RichTextFieldView";
+import { createTextField } from "../../plugin/fieldViews/TextFieldView";
+import {
+  createValidator,
+  htmlMaxLength,
+  htmlRequired,
+  maxLength,
+} from "../../plugin/helpers/validation";
+import { createReactElementSpec } from "../../renderers/react/createReactElementSpec";
+import { EmbedElementForm } from "./EmbedForm";
+
+export const createEmbedFields = () => {
```

Comment:
This shouldn't need to exist w/in a function for an embed element – the reason we do it for the imageElement is that the image select/crop functions are unique to the consumer.

So we should be fine to just export an object called `embedFields` and use that directly in the element spec.